pkgs/top-level/all-packages.nix

```diff
@@ -2702,6 +2702,7 @@ with pkgs;
     dual-function-keys =
       callPackage ../tools/inputmethods/interception-tools/dual-function-keys.nix
         { };
+    space2meta = callPackage ../tools/inputmethods/interception-tools/space2meta.nix { };
```

Comment:
You want to move your package to `pkgs/by-name`, which is the new recommended way for most packages (except some families such as python packages, ...)

Reply:
Should I also move caps2esc and dual-function-keys in this PR as well? Those plugins are already in tree so I was following that convention, but they can all move.

Reply:
I moved space2meta under pkgs/by-name.

This is a plugin for interception-tools, and the other plugins in tree don't use this convention. I can create a separate PR to move those plugins as well.